gamemodes/terrortown/gamemode/shared/hud_elements/tttwswitch/pure_skin_wswitch.lua

```diff
@@ -101,7 +109,7 @@ if CLIENT then
 		local empty_color = active and color_empty or color_empty_dark
 
 		local name = TryTranslation(wep:GetPrintName() or wep.PrintName or "...")
-		local cl1, am1 = wep:Clip1(), (wep.Ammo1 and wep:Ammo1() or false)
+		local cl1, am1, ty1 = wep:Clip1(), (wep.Ammo1 and wep:Ammo1() or false), wep.Primary.Ammo or "Pistol"
```

Comment:
In line 130 we make sure that we have a valid fallback and `ty1` isn't used anywhere else. Is there a specific reason to fallback to the "Pistol" ammo type here?